tests/test_quoting.py

```diff
@@ -57,9 +57,9 @@ def test_never_quote(quoter):
     do_not_quote = '' .join(["ABCDEFGHIJKLMNOPQRSTUVWXYZ",
                              "abcdefghijklmnopqrstuvwxyz",
                              "0123456789",
-                             "_.-"])
+                             "_.-~"])
```

Comment:
Is that in RFC?

Reply:
The change is correct.

unreserved  = ALPHA / DIGIT / "-" / "." / "_" / "~"